slack-api-model-kotlin-extension/src/main/kotlin/com/slack/api/model/kotlin_extension/block/composition/OptionGroupObjectBuilder.kt

```diff
@@ -26,6 +27,10 @@ class OptionGroupObjectBuilder private constructor(
         label = PlainTextObject(text, emoji)
     }
 
+    fun options(options: List<OptionObject>) {
```

Comment:
The tests detected lack of this method. Most people won't use this but I would like to have this for consistency.

Reply:
After merge I'll refactor to use the OptionObjectBuilder